_rules/SC4-1-2-aria-attribute-has-valid-value.md

```diff
@@ -0,0 +1,76 @@
+---
+name: ARIA attribute has valid value
+description: |
+  This rule checks that `aria-*` attributes have a valid value
+  
+success_criterion:
+- 4.1.2 # Name, Role, Value
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test procedure
+
+### Applicability
+
+Any non-empty attribute that is placed on an HTML or SVG element, and where the attribute name starts with `aria-`.
```

Comment:
This leaves a gap for attributes that aren't specified in ARIA 1.1. It's unclear if they should fail or be inapplicable.

Reply:
Rule now specifies that attributes should be listed in WAI-ARIA 1.1